com.unity.render-pipelines.core/Editor/MaterialUpgrader.cs

```diff
@@ -270,8 +270,17 @@ public void RenameKeywordToFloat(string oldName, string newName, float setVal, f
             m_KeywordFloatRename.Add(new KeywordFloatRename { keyword = oldName, property = newName, setVal = setVal, unsetVal = unsetVal });
         }
 
-        static bool IsMaterialPath(string path)
+        /// <summary>
+        /// Checking if the passed in value is a path to a Material.
+        /// </summary>
+        /// <param name="path">Path to test.</param>
+        /// <return>Returns true if the passed in value is a path to a material.</return>
+        public static bool IsMaterialPath(string path)
```

Comment:
Also same case as below this function is so simple that I'm even unsure if we should bother making it public or just use in place. Not blocking PR due to that though.

Reply:
That is correct, either it is Public, or we duplicate the code. Or we make it internal visible to somehow. Those are the options what I can see.

Reply:
Moved duplicate code into URP